docs/admin/kubeadm.md

```diff
@@ -104,17 +105,16 @@ and `--external-etcd-keyfile` flags.
 
 - `--pod-network-cidr`
 
-By default, `kubeadm init` does not set node CIDR's for pods and allows you to
-bring your own networking configuration through a CNI compatible network
-controller addon such as [Weave Net](https://github.com/weaveworks/weave-kube),
-[Calico](https://github.com/projectcalico/calico-containers/tree/master/docs/cni/kubernetes/manifests/kubeadm)
-or [Canal](https://github.com/tigera/canal/tree/master/k8s-install/kubeadm).
-If you are using a compatible cloud provider or flannel, you can specify a
-subnet to use for each pod on the cluster with the `--pod-network-cidr` flag.
-This should be a minimum of a /16 so that kubeadm is able to assign /24 subnets
-to each node in the cluster.
+For certain networking solutions the Kubernetes master can also play a role in
+allocating network ranges (CIDRs) to each node. This includes many cloud providers
+and flannel. You can specify a subnet range that will be broken down and handed out
+to each node with the `--pod-network-cidr` flag. This should be a minimum of a /16 so
+controller-manager is able to assign /24 subnets to each node in the cluster.
+If you are using flannel with [this manifest](https://github.com/coreos/flannel/blob/master/Documentation/kube-flannel.yml)
+you should use `--pod-network-cidr=10.244.0.0/16`. Most CNI based networking solutions
+do not require this flag. This includes Weave, Calico and Canal.
```

Comment:
I can just remove the `This includes Weave, Calico and Canal.` part, I'm fine with either
My primary focus is to actually document the new release, so I have no strong feelings here.
WDYT?